Sample Applications/VideoViewerDemo/MainWindow.xaml

```diff
@@ -11,7 +11,7 @@
     <!-- Setting the Directory to the relative path pointing to the Media folder.-->
     <!-- Giving this an x:Key. Now controls in this Window can bind to the videos in the Media folder.-->
     <Window.Resources>
-        <local:MyVideos Directory="../../media" x:Key="Vids" />
+        <local:MyVideos Directory="../../../media" x:Key="Vids" />
```

Comment:
This is not the write fix in my opinion, it would be better to copy the media folder to output directory and then fix the path accordingly.

Reply:
@dipeshmsft - made required changes.